src/cloud/components/WriteLimitOverlay.tsx

```diff
@@ -65,6 +96,11 @@ const WriteLimitOverlay: FC = () => {
           color={ComponentColor.Primary}
           className="rate-alert--request-increase-button"
           text="Submit Request"
```

Comment:
Same comment here re: "Submit Request" since this just opens an overlay. I would change the button to say "Contact Support."